### PR TITLE
Update README file

### PR DESCRIPTION
**Pull request by liatgo**
<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```sh
  npm install @kenshooui/react-tree@0.0.19-canary.147.400.0
  npm install @kenshooui/material-tree@0.0.2-canary.147.400.0
  # or 
  yarn add @kenshooui/react-tree@0.0.19-canary.147.400.0
  yarn add @kenshooui/material-tree@0.0.2-canary.147.400.0
  ```
</details>

### CODE REVIEW

File: packages/core/src/tree.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ const Tree = props => {
     onSelect,
     width,
     height,
+    styles,
```
Comment (liatgo) [inline packages/core/src/tree.js:28]: @liorheber In the previous PRs I removed "styles" and "classname" props bacuse I thought They are not in use.
Noe when writing the README I understood the "styles" is needed for customized styles.
What about calssname? Do we need this property?

File: README.md [hunk 2]
```diff
@@ -31,12 +57,221 @@ const structure = [
...
+ 
+  <p align="center">
+      <img src="react-tree-demo.gif?raw=true" width="400" />
+  </p>
+  
+  ### Installation
+  
+   **Installation using npm:**
+  ```
+   npm install @kenshooui/material-tree --save
```
Comment (liorheber) [inline README.md:70]: You need react-tree as well as it's a peerDependency

File: README.md [hunk 3]
```diff
@@ -31,12 +57,221 @@ const structure = [
...
+
+```
+ yarn add @kenshooui/material-tree
+```
+
+ ### How to use
+ <!-- example -->
+
+```jsx
+import MaterialTree from "@kenshooui/material-tree";
```
Comment (liorheber) [inline README.md:82]: This is the root readme, it should not contain the readme for the material design.
You have the root readme that should discuss the core component and link to a readme for the different themes.
In those specific theme readme you can show information on how to use the theme.

File: packages/core/src/tree.js (hunk 1 above)
Comment (liorheber) [reply packages/core/src/tree.js:28]: Which classNames? Can you link the change?
You would usually need the className prop to customize using almost any styling library - css modules, styled-components etc.

File: README.md (hunk 2 above)
Comment (liorheber) [reply README.md:70]: Also material core and icons (add a comment that they are needed if they don't exist in the project yet)

File: packages/core/src/tree.js (hunk 1 above)
Comment (liatgo) [reply packages/core/src/tree.js:28]: https://github.com/kenshoo/react-tree/pull/143/files
@liorheber I tagged you in the PR

File: README.md (hunk 3 above)
Comment (liatgo) [reply README.md:82]: @liorheber What about the core readme?
Should I describe all the information related to it in the root readme of the project? or in the inner readme of the core package?

File: packages/core/src/tree.js (hunk 1 above)
Comment (liorheber) [reply packages/core/src/tree.js:28]: They are for customization, however, I'm not sure it was fully implemented.
I suggest returning it and we can have a look at it at a later task when we decide to support that type of customization.

File: packages/material_tree/README.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+  
+   **Installation using npm:**
+  ```
+   npm install @kenshooui/material-tree --save
+
+   npm install @kenshooui/react-tree --save
+
+   npm install @material-ui/core --save
+
+   npm install @material-ui/icons --save
```
Comment (liorheber) [inline packages/material_tree/README.md:26]: You can do this, and the yarn command in a single line - 
npm install --save @material-ui/core @material-ui/icons @kenshooui/react-tree @kenshooui/material-tree
yarn add @material-ui/core @material-ui/icons @kenshooui/react-tree @kenshooui/material-tree

Also worth mentioning that the material dependencies are only if they don't exist already in the app.

**Comment by liorheber** [comment]
Merging this, please fix my comments in another PR.
When you issue the PR for it, there's a label section in Github, use the skip-release label so we can avoid releasing a new version to npm for the documentation.

**Comment by liorheber** [comment]
You also need to place the API documentation of the core component in the core package.
The root should be the link to the two.

File: packages/material_tree/README.md (hunk 4 above)
Comment (liatgo) [reply packages/material_tree/README.md:26]: @liorheber isn't this enough: "In addition, dependencies to <b>material-ui icons and core components</b> are needed if they don't exist in your project." ?
Do you want me to rephrase it?

**Comment by liatgo** [comment]
> Merging this, please fix my comments in another PR.
> When you issue the PR for it, there's a label section in Github, use the skip-release label so we can avoid releasing a new version to npm for the documentation.

@liorheber This PR included few changes in the components as well. Do we want to avoid releasing in this case?

**Pull request merged**